### ISSUE TITLE
doesn't work.. downloads a .htm file along with a .exe file

### ISSUE THREAD

**Issue by shmizan** [Ede123/userscripts#2]
doesn't work.. downloads a .htm file along with a .exe file

**Comment by Ede123** [comment]
Do you mind telling me which userscript this is about?

**Comment by shmizan** [comment]
sorry. I meant the opensubtitles direct download one.

**Comment by Ede123** [comment]
OK thanks.

Are you using the most recent version? I just recently pushed version 1.1 (see 74612874019fad699bd65d9fb8d6e15a32c28217) which fixed such an issue.

Seems OpenSubtitles is now actively trying to prevent direct downloads, but a quick test just now worked fine for me...

**Comment by Ede123** [comment]
Oh, and are you using an add-blocker?
OpenSuptitles recently added some nasty advertisement that sometimes will open when clicking anywhere on the page but that is usually suppressed by add-blockers (e.g. uBlock origin).

**Comment by shmizan** [comment]
Hi, sorry for being vague. 
I will try to be as clear as I can:
1. I didn't see the update.
2. after using the updated userscript, going into the webpage of a specific subtitle, where you can see the "thanks" and "download" button, clicking download is directly downloading.
3. when viewing a list of subtitles, and you see a chart with the download column, clicking the download button (that button that says for example - "63x srt") is sending you to that "You will get subtitles from OpenSubtitles.org soon. Support us and get instant download" page. I thought it was suppose to work there too?

thanks!!

**Comment by Ede123** [comment]
No, actually it's only designed to work with the "Download" button.

I checked if it is possible to adjust the script for those links, too, but there is no good way. As I can't extract the direct download link from the page the only possibility would be to follow the link (i.e. open the unwanted page in background) and extract the actual location of the subtitle file from there.

I'll consider this for the future but right now I suggest to work around the issue by expanding the entry in the list and use the "Download" button there...
